app/javascript/elm/tests/MainTests.elm

```diff
@@ -66,24 +44,14 @@ popups =
         ]
 
 
-sensors : List Sensor.Sensor
-sensors =
-    [ { parameter = "parameter"
-      , name = "Sensor"
-      , unit = "unit"
-      , session_count = 1
-      }
-    ]
-
-
 parameterSensorFilter : Test
 parameterSensorFilter =
     describe "Parameter filter tests: "
         [ test "parameter filter shows parameter name based on selectedSensorId" <|
             \_ ->
                 { defaultModel
                     | selectedSensorId = "parameter-sensor (unit)"
-                    , sensors = sensors
+                    , sensors = defaultSensors
```

Comment:
Maybe we can have a `defaultModel` that already contains `defaultSensors` in TestUtils.elm. Would make these test cases less noisy.